packages/compiler/core/checker.ts

```diff
@@ -3042,7 +3082,7 @@ export function createChecker(program: Program): Checker {
   }
 }
 
-function isErrorType(type: Type): type is ErrorType {
+export function isErrorType(type: Type): type is ErrorType {
```

Comment:
Seems like we have this helper method already in `decorators.ts along side with isVoidType, isNeverType, should we move them all in the checker or into a new helper file?

Reply:
Probably. I didn't end up needing this so I'll revert it to keep the PR on task.

Reply:
That said, I'm almost sure I've missed some things here, I'll see if I can run this over all the identifiers in our tests.